client/gulpfile.js

```diff
@@ -28,6 +29,14 @@ const sources = {
     distPath: '../dist/assets/js/',
     all: 'js/**/*.js'
   },
+  jsHead: {
+    entry: './js/jsHead.js',
+    distPath: '../server/views/_partials/'
```

Comment:
We should never be building into the server folder - could we have this into the dist folder?